rke/k8s_defaults.go

```diff
@@ -63,12 +64,80 @@ func initData() {
 	DriverData.CisConfigParams = loadCisConfigParams()
 	DriverData.CisBenchmarkVersionInfo = loadCisBenchmarkVersionInfo()
 
-	if err := readFile("./channels.yaml", DriverData.K3S); err != nil {
+	loadChannelInfo()
+}
+
+func loadChannelInfo() {
+	// closures process the data, allowing us to reuse readFiles in different contexts
```

Comment:
It appears as though you're providing a doc comment on what the function is doing and responsible for. Please move this to standard Go doc comment style. https://gist.github.com/adamveld12/c0d9f0d5f0e1fba1e551#doc-comments

Reply:
I removed this comment since it wasn't necessary.